libs/src/services/creatorNode/index.js

```diff
@@ -75,7 +75,31 @@ class CreatorNode {
 
   /* -------------- */
 
-  constructor (web3Manager, creatorNodeEndpoint, isServer, userStateManager, lazyConnect, schemas) {
+  /**
+   * Constructs a service class for a creator node
+   * @param {Web3Manager} web3Manager
+   * @param {string} creatorNodeEndpoint fallback creator node endpoint (to be deprecated)
+   * @param {boolean} isServer
+   * @param {UserStateManager} userStateManagern  singleton UserStateManager instance
+   * @param {boolean} lazyConnect whether or not to lazy connect (sign in) on load
+   * @param {*} schemas
+   * @param {Set<string>?} passList whether or not to include only specified nodes (default null)
+   * @param {Set<string>?} blockList whether or not to exclude any nodes (default null)
+   * @param {object?} monitoringCallbacks callbacks to be invoked with metrics from requests sent to a service
+   *    @param {function} monitoringCallbacks.request
```

Comment:
nit - theres still some extra spacing `*    @`

Reply:
this is nested on purpose i thought it made it more clear, disagree?

Reply:
I don't see that style across the repo so I would go against it, but I am ok with either!

Reply:
sure, ill remove!